app/globals.css

```
@@ -102,6 +102,8 @@ button:focus-visible, a:focus-visible { outline: 2px solid var(--ink); outline-o
 .material-file__path { max-width: 100%; overflow: hidden; color: var(--index-faint); font: 10px/1.2 var(--mono); text-overflow: ellipsis; white-space: nowrap; unicode-bidi: plaintext; }
 .material-file__structure-control,
 .material-file__context-space { position: relative; z-index: 1; display: grid; width: var(--material-file-action-width); height: var(--index-row); place-items: center; padding: 0; border: 0; background: transparent; color: inherit; }
+.material-file__leaf-marker { pointer-events: none; }
+.material-file__leaf-marker::before { width: 3px; height: 3px; border-radius: 50%; background: currentColor; content: ""; opacity: .42; }
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)